src/ipred.rs

```diff
@@ -155,12 +156,11 @@ impl pal_pred::Fn {
 
 #[repr(C)]
 pub struct Rav1dIntraPredDSPContext {
-    // TODO(legare): Remove `Option` once `dav1d_submit_frame` is no longer checking
-    // this field with `is_none`.
-    pub intra_pred: [Option<angular_ipred::Fn>; 14],
+    pub intra_pred: [angular_ipred::Fn; 14],
     pub cfl_ac: [cfl_ac::Fn; 3],
     pub cfl_pred: [cfl_pred::Fn; 6],
     pub pal_pred: pal_pred::Fn,
+    pub initialized: bool,
```

Comment:
See other comment, but should this move to the `DSPContext`?

Reply:
Changed in [7d0c2ce..4609590](https://github.com/memorysafety/rav1d/compare/7d0c2ce6da7544f7a9f026d1b44478f4ac19b5ff..4609590678b588824c7770f5d05ba378e2914709).